apps/files/tests/test_models.py

```diff
@@ -806,6 +806,40 @@ def test_limit_validator_errors(self):
         assert validation['messages'][0]['type'] == 'error'
 
 
+def test_file_upload_passed_auto_validation_passed():
+    upload = FileUpload(validation=json.dumps({
+        'passed_auto_validation': True,
```

Comment:
could you use a copy of [VALIDATOR_SKELETON_RESULTS](https://github.com/mozilla/olympia/blob/master/apps/constants/base.py#L372) before adding this attribute? Otherwise, the test may be prone to false positives triggered by incomplete validation.